docs/production-deployment/cloud/gcp-export-gcs.mdx

```diff
@@ -0,0 +1,166 @@
+---
+id: gcp-export-gcs
+title: Exporting Workflow Event History to GCS
+sidebar_label: GCP Export to GCS
+description: Export Workflow History to GCS
+slug: /cloud/export/gcp-export-gcs
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - verify
+  - workflow history export
+  - gcp
+tags:
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - verify
+  - workflow-history-export
+  - gcp
+---
+
+## **Prerequisites**
+
+Before configuring the Export sink, please complete the following steps in Google Cloud.
+
+1. Create a GCS bucket and take note of its bucket name, for example, "test-export"
+    1. If you wish to enable customer-managed encryption keys (CMEK), do so.
+2. Record the GCP Project ID that owns the bucket.
+3. Set up a service account in the same project that trusts the Temporal internal service account to let Temporal write information to your account.
+4. Follow the instructions in the Temporal Cloud UI, there are two ways to set up this service account:
```

Comment:
This heirachy is fine but probably could be made a little more clearer. Would this be accurate? 
"Follow the instructions in the Temporal Cloud UI to set up the service account. You have two options:

Manual Setup:
1. Input the service account ID, GCP project ID, and GCS bucket name as prompted.
2. Follow the on-screen instructions to manually create a new service account.

Automated Setup:
1. Use the provided [Terraform template](https://github.com/temporalio/terraform-modules/tree/main/modules/gcp-sink-sa) to automatically create the service account.""

Reply:
Sure thing.